Makefile

```diff
@@ -7,7 +7,7 @@ docker_opts ?= --rm --tty --user "$$(id -u)"
 
 antora_cmd  ?= $(docker_cmd) run $(docker_opts) --volume "$${PWD}":/antora vshn/antora:3.0.0
 antora_opts ?= --cache-dir=.cache/antora
-preview_cmd ?= $(docker_cmd) run --rm --publish 35729:35729 --publish 2020:2020 --volume "${PWD}":/preview/antora vshn/antora-preview:3.1.1.1 --antora=docs --style=appuio
+preview_cmd ?= $(docker_cmd) run --rm --publish 35729:35729 --publish 2020:2020 --volume "${PWD}":/preview/antora ghcr.io/vshn/antora-preview:master --antora=docs --style=appuio
```

Comment:
Do not use the `master` image, always use the latest released version. See https://github.com/vshn/antora-preview/releases

Reply:
I changed the image v3.1.4